store/storetest/post_store.go

```diff
@@ -93,9 +92,8 @@ func testPostStoreSaveChannelMsgCounts(t *testing.T, ss store.Store) {
 	o1.Type = model.POST_REMOVE_FROM_TEAM
 	require.Nil(t, (<-ss.Post().Save(&o1)).Err)
 
-	res = <-ss.Channel().Get(c1.Id, false)
+	c1, err = ss.Channel().Get(c1.Id, false)
 	require.Nil(t, res.Err)
```

Comment:
The `require` needs to check for `err`, `res` belongs to the channel save:

```suggestion
	require.Nil(t, err)
```

Reply:
No problem, thanks for the review!